spec/integration/primitives_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'spec_helper'
+require 'time'
+
+describe "chef-provisioning-docker" do
+  extend DockerSupport
+
+  when_the_chef_12_server "exists", organization: 'foo', server_scope: :context do
+
+    let(:ubuntu_options) {
+      {
+        :base_image => {
+          :name => 'ubuntu',
+          :repository => 'ubuntu',
+          :tag => '14.04'
+        },
+      }
+    }
+
+    let(:iso_date) { Time.now.iso8601.gsub(':', '') }
+
+    context "machine_image resource" do
+      [:create].each do |test_action|
+        it ":#{test_action} succeeds" do
+          options = ubuntu_options
+          date = iso_date
+
+          expect_recipe {
+            with_driver 'docker'
```

Comment:
We should be able to add this globally into the DockerSupport - check out https://github.com/chef/chef-provisioning-aws/blob/master/spec/aws_support.rb#L74